### ISSUE TITLE
The app needs an icon

### ISSUE THREAD

**Issue by lealgo** [fohristiwhirl/nibbler#47]
A dock is sad without it.

**Comment by fohristiwhirl** [comment]
Heh. Does it not get the Electron icon when running?

**Comment by lealgo** [comment]
> Heh. Does it not get the Electron icon when running?

Not on Linux (Ubuntu 18.04 Gnome).

**Comment by lealgo** [comment]
But it deserves a better icon than the Electron one...

**Comment by fohristiwhirl** [comment]
By the way, on an unrelated matter, do you ever experience [this bug](https://github.com/fohristiwhirl/nibbler/issues/24)?

**Comment by lealgo** [comment]
> By the way, on an unrelated matter, do you ever experience [this bug](https://github.com/fohristiwhirl/nibbler/issues/24)?

Yes, although it's very subtle. I haven't noticed myself. But I got the impression that the piece itself doesn't flash, it's the square. Maybe the select square action is interfeering?

**Comment by AlexisOlson** [comment]
<img src="https://img.icons8.com/color/144/000000/futurama-nibbler.png"/>

https://icons8.com/icon/tyoFz4DDW1cr/futurama-nibbler

<img src="https://vectorified.com/image/futurama-vector-22.png"/>

https://vectorified.com/download-image#futurama-vector-22.png

<img src="http://icons.veryicon.com/png/Movie%20%26%20TV/Futurama%20Vol.%201/Nibbler.png"/>

http://www.veryicon.com/icons/movie--tv/futurama-vol-1/nibbler.html

**Comment by lealgo** [comment]
I love the last one, I use it on my launcher.

**Comment by CiriousJoker** [comment]
oh plz no :(
If it's from futurama, it's definitely copyrighted, even if it's altered. The reason why it "looks good" (even if I think it's ugly) is because of their original work.

Here's my attempt at creating an icon. It's currently just a draft, but if you like the idea, I can polish it up a little and push it.

![Nibbler AppIcon](https://user-images.githubusercontent.com/16866223/87366667-86b5d680-c579-11ea-94de-cb6fabf2f3c3.png)

The icon contains this icon: https://www.svgrepo.com/svg/155301/chess, which is licensed under CC0

**Comment by ciriousjoker** [comment]
I was happy to see that this mac version fork is using my icon suggestion. I'd be even more happy if the base version would also switch ;)

**Comment by Hypenate** [comment]
I think we can close this? :D
